### PR TITLE
Friendly_ID

### PR DESCRIPTION
**Pull request by kamok**
Ok so if I select base to be your master, I won't submit pull request to YOUR friendly_id branch. 

Kk.

### CODE REVIEW

File: app/models/topic.rb [hunk 1]
```diff
@@ -1,2 +1,4 @@
 class Topic < ActiveRecord::Base
+   extend FriendlyId
```
Comment (dylanerichards) [inline app/models/topic.rb]: Indentation is inconsistent.

File: config/initializers/friendly_id.rb [hunk 2]
```diff
@@ -0,0 +1,88 @@
+# FriendlyId Global Configuration
+#
```
Comment (dylanerichards) [inline config/initializers/friendly_id.rb]: Please remove boilerplate comments.

**Comment by dylanerichards** [comment]
Cant merge this PR just yet, as tests haven't been provided.

**Comment by dylanerichards** [comment]
Great job breaking actions into separate commits.

File: app/controllers/topics_controller.rb [hunk 3]
```diff
@@ -8,15 +8,15 @@ def new
   end
 
   def show
```
Comment (dylanerichards) [inline app/controllers/topics_controller.rb]: Please refactor `@topic = Topic.friendly.find(params[:id])` into a `private` `#set_topic` `before_action` that only runs before the methods that use it.

Comment (kamok) [inline app/controllers/topics_controller.rb]: https://gist.github.com/kamok/d3feb8976b364b570ecf951a7c11cfc5

**Pull request closed without merging**